### PR TITLE
Lesson 2 - Exercises

### PR DESCRIPTION
**Pull request by aguardia**
I built three functions with the different goals

### CODE REVIEW

**Comment by aguardia** [comment]
Added the fibonacci and Lucas sequence exercise

File: students/AleGuardia/lesson2/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+for i in range(1,101):
+    word = ""
+    if i % 3 == 0 or i % 5 == 0:
+        if i % 3 == 0:
+            word += "Fizz"
+        if i % 5 == 0:
+            word += "Buzz"
+        print(word)
+        continue
```
Comment (charlesrobison) [inline students/AleGuardia/lesson2/FizzBuzz.py]: continue is not necessary here.  Solution works well.

File: students/AleGuardia/lesson2/print_grid.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    for i in range(number):
+        print(row)
+        print()
+        for j in range(units):
+            print(column)
+            print()
+    print(row)
+
+
+print_grid_custom_two(5,3)
```
Comment (charlesrobison) [inline students/AleGuardia/lesson2/print_grid.py]: Works well - nice job.

File: students/AleGuardia/lesson2/series.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        print("The expected " + str(i) + " element in the " + name + " sequence is: " + str(series[i]))
+        print("The result from the function is " + str(sum_series(i,2,1)))
+        if sum_series(i,2,1) == series[i]:
+            print("Test Passed")
+        else:
+            print("Test Failed)")
+
+
+test_series(fib_series,"Fibonacci")
+test_series(luc_series,"Lucas")
```
Comment (charlesrobison) [inline students/AleGuardia/lesson2/series.py]: Tests look good but fibonacci and lucas functions don't seem to be functioning as expected.  Recommend reviewing assignment requirements.  Should be able to run each function and return expected results - e.g., fibonacci(4) should return 3 etc.

**Pull request merged**